frontend/src/layout/navigation-3000/sidepanel/panels/sidePanelDocsLogic.ts

```diff
@@ -14,32 +29,58 @@ export const sidePanelDocsLogic = kea<sidePanelDocsLogicType>([
 
     actions({
         openDocsPage: (urlOrPath: string) => ({ urlOrPath }),
+        updatePath: (path: string) => ({ path }),
+        setInitialPath: (path: string) => ({ path }),
+        unmountIframe: true,
+        handleExternalUrl: (urlOrPath: string) => ({ urlOrPath }),
     }),
 
     reducers(() => ({
-        path: [
+        currentPath: [
+            null as string | null,
+            {
+                updatePath: (_, { path }) => sanitizePath(path),
+            },
+        ],
+        initialPath: [
             '/docs' as string,
+            { persist: true },
```

Comment:
Should this be remembered across page loads of different in-app URLs? I was rather thinking that the docs would be contextual, i.e. `docsPath` would be part of `SceneConfig`, and we'd show the relevant docs page on docs panel open

Reply:
I think we still need to play with it. Realistically if you close the doc and open again it would be annoying to "lose" your place.

For now I'm going to leave it and then once we start adding support for scene-aware docs then we can figure out what to do (maybe we cache it for an amount of time before resetting to be based on the scene